src/test/java/com/fullteaching/e2e/no_elastest/common/EdgeUser.java

```diff
@@ -51,42 +50,31 @@ public EdgeUser(String userName, int timeOfWaitInSeconds, String testName, Strin
         if (eusApiURL == null) {
             this.driver = new EdgeDriver(options);
         } else {
-            try {
                 Map<String, Object> selenoidOptions = new HashMap<>();
                 log.info("Using the remote WebDriver (Selenoid)");
-                SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd_HH:mm");
-                log.debug("Adding all the extra capabilities needed: {testName,enableVideo,enableVNC,name,enableLog,videoName,screenResolution}");
-
-                selenoidOptions.put("testName", testName + "_" + userIdentifier + "_" + format.format(new Date()));
-                //CAPABILITIES FOR SELENOID RETORCH
-                selenoidOptions.put("enableVideo", true);
-                selenoidOptions.put("enableVNC", true);
-                selenoidOptions.put("name", testName + "_" + userIdentifier);
 
                 DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yy-MM-dd-HH:mm");
                 LocalDateTime now = LocalDateTime.now();
-                String logName = dtf.format(now) + "-" + testName + "-" + userIdentifier + ".log";
-                String videoName = dtf.format(now) + "_" + testName + "_" + userIdentifier + ".mp4";
-                log.debug("The data of this test would be stored into: video name " + videoName + " and the log is " + logName);
+                String baseName = System.getProperty("tjob_name") + "-" + dtf.format(now) + "-" + testName + "-" + userIdentifier ;
+                log.debug("The data of this test are stored into .mp4 and .log files named: {}" , baseName);
+                log.debug("Adding all the extra capabilities needed: {testName,enableVideo,enableVNC,name,enableLog,videoName,screenResolution}");
+                //CAPABILITIES FOR SELENOID
 
+                selenoidOptions.put("testName", testName + "-" + userIdentifier + "-" + dtf.format(now));
+                selenoidOptions.put("enableVideo", true);
+                selenoidOptions.put("enableVNC", true);
+                selenoidOptions.put("name", testName + "-" + userIdentifier);
                 selenoidOptions.put("enableLog", true);
-                selenoidOptions.put("logName ", logName);
-                selenoidOptions.put("videoName", videoName);
-
+                selenoidOptions.put("logName ", String.format("%s%s",baseName.replaceAll("\\s","") , ".log"));
+                selenoidOptions.put("videoName", String.format("%s%s",baseName.replaceAll("\\s","") ,".mp4"));
                 selenoidOptions.put("screenResolution", "1920x1080x24");
-
                 options.setCapability("selenoid:options", selenoidOptions);
-
                 //END CAPABILITIES FOR SELENOID RETORCH
```

Comment:
This part of selenoid options seems to appear three times, it should be refactored, perhaps in the base class.
Do not do this now, create a new issue for it and annotate the number by replying this comment.

Note: Maybe there are more lines of code that are repeated, e.g. before this commented block.

Reply:
Done, created issue #146 with a associated branch from this PR

Reply:
Addressed in  https://github.com/giis-uniovi/retorch-st-fullteaching/tree/146-reduce-the-duplicated-code-in-base-chrome-edge-firefoxuser